### PR TITLE
Final (? Front page) Update

### PR DESCRIPTION
**Pull request by gburnhill**
Incorporated Sion requests
Fixed sticky-nav to be nicer
Sorted a few other mild annoyances

### CODE REVIEW

File: HelpMyStreetFE/HelpMyStreetFE/src/js/shared/cookie-helper.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
         // - description: Description for this cookie type. Displayed in
         //                title attribute.        
         title: 'This website uses cookies',
-        message: ' We use cookies to help make our website usable and to understand how visitors interact with us. More information can be found in our <a href="/privacy-policy" target="_blank">Privacy Policy</a>. <br /> You can change or withdraw your consent at any time by changing your browser settings.',
+        message: ' We use cookies to help make our website usable and to understand how visitors interact with us. More information can be found in our <a href="/privacy-policy" target="_blank">Privacy Policy</a>. <br /> You can change or disalbe cookies at any time by changing your browser settings.',
```
Comment (usmaanrazaq) [inline HelpMyStreetFE/HelpMyStreetFE/src/js/shared/cookie-helper.js:20]: typo on disable

**Comment by gburnhill** [comment]
Well noticed!

Sorted

G

On 1 Jun 2020, at 16:17, usmaanrazaq <notifications@github.com<mailto:notifications@github.com>> wrote:



@usmaanrazaq requested changes on this pull request.

________________________________

In HelpMyStreetFE/HelpMyStreetFE/src/js/shared/cookie-helper.js<https://github.com/HelpMyStreet/front-end/pull/244#discussion_r433297032>:

> @@ -17,7 +17,7 @@
         // - description: Description for this cookie type. Displayed in
         //                title attribute.
         title: 'This website uses cookies',
-        message: ' We use cookies to help make our website usable and to understand how visitors interact with us. More information can be found in our <a href="/privacy-policy" target="_blank">Privacy Policy</a>. <br /> You can change or withdraw your consent at any time by changing your browser settings.',
+        message: ' We use cookies to help make our website usable and to understand how visitors interact with us. More information can be found in our <a href="/privacy-policy" target="_blank">Privacy Policy</a>. <br /> You can change or disalbe cookies at any time by changing your browser settings.',


typo on disable

**Comment by mike-f50** [comment]
@usmaanrazaq @gburnhill looks like there's disagreement on the definition of a small button.  Any comments?

`    &.small {
<<<<<<< HPR-2
        @media screen and (max-width: $sm-screen) {
        border-radius: 12px;
        font-size: 0.9rem;
        padding: 8px 14px;
        line-height: 1;
=======
        border-radius: 16px;
        padding: 10px 10px;
        font-size: 0.7rem;

        @media screen and (max-width: $sm-screen) {
            width: 100%;
>>>>>>> next-major-release
        }
    }`

**Comment by mike-f50** [comment]
Apologies for the creative formatting there!

**Comment by gburnhill** [comment]
I mean obviously I think MY small button is the most important and correct kind...

Not sure where else small buttons appear… or why there’s a disagreement….

I’m sure I added the small button class so I don’t know why it’s appeared in NMR…

Yours unhelpfully and much confused
G

On 1 Jun 2020, at 16:50, mike-f50 <notifications@github.com<mailto:notifications@github.com>> wrote:



Apologies for the creative formatting there!

**Comment by usmaanrazaq** [comment]
It looks like the small button class was introduced in NMR for the cookie consent popup buttons. The styles don't look too different

**Comment by usmaanrazaq** [comment]
it looks like the other .small class was introduced into NMR by the Cookie Consent changes. I've sorted the conflict by having @gburnhill code on the small class, as its more likely to be used by the rest of the site. 

I've placed the other small class thats needed for the cookie popup in the cookie-consent.scss  file.

**Pull request merged**